<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganisateurInscriptionFrame</class>
 <widget class="QFrame" name="OrganisateurInscriptionFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>1431</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Acte de naissance :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_voir_acte_naissance_3">
       <property name="text">
        <string>voir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>301</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>CV :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_voir_cv_3">
       <property name="text">
        <string>voir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_accepte_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Validé</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Diplôme:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_voir_diplome_3">
       <property name="text">
        <string>voir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_visionneuse_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>-40</y>
      <width>500</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>301</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Prénom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="champ_prenom_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_retour_2">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>530</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour à la liste</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>301</width>
      <height>19</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="champ_nom_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_rejete_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Rejeté</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>301</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Photo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_voir_photo">
       <property name="text">
        <string>Voir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_6</zorder>
   <zorder>layoutWidget_7</zorder>
   <zorder>btn_accepte_2</zorder>
   <zorder>layoutWidget_8</zorder>
   <zorder>label_visionneuse_2</zorder>
   <zorder>layoutWidget_9</zorder>
   <zorder>btn_retour_2</zorder>
   <zorder>layoutWidget_10</zorder>
   <zorder>btn_rejete_2</zorder>
  </widget>
  <widget class="QTableView" name="table_inscriptions">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>1300</width>
     <height>630</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>table_inscriptions</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>OrganisateurInscriptionFrame</receiver>
   <slot>onTableCellClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>-33</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTableCellClicked(QModelIndex)</slot>
 </slots>
</ui>
